### PR TITLE
Update addresses console component

### PR DESCRIPTION
**Pull request by Brentdevent**
Address updates to fix console component.

### CODE REVIEW

**Review by momo5502 — APPROVED** [review]

**Comment by momo5502** [comment]
How to we keep track of what got updated and what not?

File: src/client/component/console.cpp [hunk 1]
```diff
@@ -251,7 +251,7 @@ namespace console
 			{
 				{
 					static utils::hook::detour sys_create_console_hook;
-					sys_create_console_hook.create(game::select(0x1423339C0, 0x140597880), sys_create_console_stub);
+					sys_create_console_hook.create(game::select(0x142332E00, 0x140597880), sys_create_console_stub);
```
Comment (momo5502) [inline src/client/component/console.cpp:254]: 0x1423333C0 is the correct address

Comment (Brentdevent) [reply src/client/component/console.cpp:254]: isn't Sys_ShowConsole at address 0x1423333C0?

**Comment by Brentdevent** [comment]
> How to we keep track of what got updated and what not?

Good question, maybe we should do something like a checklist, taskboard?
Could also be useful to keep track of who is doing what so we're not colliding.

File: src/client/component/console.cpp (hunk 1 above)
Comment (momo5502) [reply src/client/component/console.cpp:254]: jup, iwas dumb

**Comment by momo5502** [comment]
> > How to we keep track of what got updated and what not?
> 
> Good question, maybe we should do something like a checklist, taskboard? Could also be useful to keep track of who is doing what so we're not colliding.

imo, the best thing to do is update all addresses in symbols.hpp, and then each component can be done individually

**Comment by momo5502** [comment]
components for now are easy to distinguish with the REGISTER_COMPONENT_WORKING macro

**Comment by Brentdevent** [comment]
Aah yes, right 👍🏼

**Pull request merged**